systemtests/src/test/java/io/enmasse/systemtest/bases/plans/PlansTestBase.java

```diff
@@ -131,6 +144,65 @@ public void doTestUnknownAddressPlan(AddressSpace addressSpace, List<StageHolder
         } while(true);
     }
 
+    public void doTestUpdatePlanBrokerCredit_changesPerAddressMaxSize(AddressSpace addressSpace, Address queueDest, AddressPlan phase1, AddressPlan phase2, AddressPlan redefinedPhase2, AmqpClientFactory amqpClientFactory) throws Exception {
```

Comment:
Please use java camel syntax instead of `_`

Reply:
Fixed by @vbusch